code/tests/utilities/test_LangChainAgent.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import MagicMock
+
+from backend.batch.utilities.orchestrator.LangChainAgent import LangChainAgent
+from backend.batch.utilities.common.Answer import Answer
+
+
+class Testing_LangChainAgent(LangChainAgent):
+    def __init__(self) -> None:
```

Comment:
Nice way of overriding values when you can't provide them via the constructor! Do we need to call `super().__init__()` here though? I can see we're doing it in a few places, including the `LangChainAgent`,

Reply:
If I call super.__init then I will also create a whole set of connected server objects which makes it hard to test. The idea was to not initialise the base class and just to test these two methods.